libvmaf/src/feature/funque_adm.c

```diff
@@ -0,0 +1,319 @@
+/**
```

Comment:
code related to FUNQUE should be place in `libvmaf/src/feature/third_party/funque/` to keep with the VMAF contribution guide for 3rd party feature extractors

Reply:
All the files related to FUNQUE are moved to libvmaf/src/feature/third_party/funque